### PR TITLE
e2image: add page

### PR DESCRIPTION
**Pull request by deeeeeeps**
Page requested #4250

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/e2image.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# e2image
+
+> Save critical ext2/ext3/ext4 filesystem metadata to a file.
+> More information: <https://linux.die.net/man/8/e2image>.
+
+- Save metadata located on device to the specified file:
+
+`e2image {{/dev/sdXN}} {{path/to/image-file}}`
```
Comment (navarroaxel) [inline pages/linux/e2image.md:8]: The variable names should be in `snake_case`, so it should be `image_file` but I think just `file` is enough. What do you think?
```suggestion
`e2image {{/dev/sdXN}} {{path/to/file}}`
```

File: pages/linux/e2image.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# e2image
+
+> Save critical ext2/ext3/ext4 filesystem metadata to a file.
+> More information: <https://linux.die.net/man/8/e2image>.
+
+- Save metadata located on device to the specified file:
```
Comment (navarroaxel) [inline pages/linux/e2image.md:6]:
```suggestion
- Write metadata located on device to a specific file:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @deeeeeeps, welcome and thanks for your submission. I left 2 comments.

File: pages/linux/e2image.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`e2image {{/dev/sdXN}} {{path/to/file}}`
+
+- Print metadata located on device to `stdout`:
+
+`e2image {{/dev/sdXN}} -`
+
+- Restore the filesystem metadata back to the device:
+
+`e2image -I {{/dev/sdXN}} {{path/to/image-file}}`
```
Comment (navarroaxel) [inline pages/linux/e2image.md:16]:
```suggestion
`e2image -I {{/dev/sdXN}} {{path/to/file}}`
```

File: pages/linux/e2image.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+- Create a large raw sparse file with metadata at proper offsets:
+
+`e2image -r {{/dev/sdXN}} {{path/to/image-file}}`
```
Comment (navarroaxel) [inline pages/linux/e2image.md:20]:
```suggestion
`e2image -r {{/dev/sdXN}} {{path/to/file}}`
```

File: pages/linux/e2image.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- Ceate a QCOW2 image file instead of a normal or raw image file:
+
+`e2image -Q {{/dev/sdXN}} {{path/to/image-file}}`
```
Comment (navarroaxel) [inline pages/linux/e2image.md:24]:
```suggestion
`e2image -Q {{/dev/sdXN}} {{path/to/file}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just a few more changes.

File: pages/linux/e2image.md (hunk 5 above)
Comment (sbrl) [reply pages/linux/e2image.md:24]: Or even 
```suggestion
`e2image -Q {{/dev/sdXN}} {{path/to/image_file}}`
```

File: pages/linux/e2image.md (hunk 3 above)
Comment (sbrl) [reply pages/linux/e2image.md:16]: Or even
```suggestion
`e2image -I {{/dev/sdXN}} {{path/to/image_file}}`
```

File: pages/linux/e2image.md (hunk 4 above)
Comment (sbrl) [reply pages/linux/e2image.md:20]: Or even
```suggestion
`e2image -r {{/dev/sdXN}} {{path/to/image_file}}`
```

File: pages/linux/e2image.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# e2image
+
+> Save critical ext2/ext3/ext4 filesystem metadata to a file.
+> More information: <https://linux.die.net/man/8/e2image>.
+
+- Write metadata located on device to a specific file:
+
+`e2image {{/dev/sdXN}} {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/linux/e2image.md:8]:
```suggestion
`e2image {{/dev/sdXN}} {{path/to/image_file}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @deeeeeeps!

**Pull request merged**